lib/accessors/regexp.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.exports = function asRegExp (value, flags) {
+  // We have to test the value and flags indivudally if we want to write our
+  // own error messages,as there is no way to differentiate between the two
+  // errors except by using string comparisons.
+
+  // Test the flags
+  try {
```

Comment:
Yeah I think this is the best call. Plus I don't think performance is a huge concern since you'd usually read environment variables in once at startup.